index.html

```diff
@@ -5,15 +5,90 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>AJAX Project</title>
   <link rel="icon" href="images/favicon.ico">
+  <link rel="stylesheet" crossorigin="anonymous"
+    href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.14.0/css/all.min.css"
+    integrity="sha512-1PKOgIY59xJ8Co8+NE6FZ+LOAZKjy+KY8iq0G4B3CyeY6wYHN3yt9PW0XpSriVlkMXe40PTKnXrLnZ9+fkDaog==">
   <link rel="stylesheet" href="css/reset.css">
   <link rel="stylesheet" href="css/layout.css">
   <link rel="stylesheet" href="css/styles.css">
 </head>
 <body>
-  <header>
+  <header class="font-family-proza">
+    <div class="container navBar">
+      <div class="row">
+        <div class="margin-right-30px align-content-center">
+          <h2 class="font-family-proza">Stock Watch</h2>
+        </div>
+      </div>
+    </div>
   </header>
   <main>
+    <div class="container">
+
+      <div class="row align-content-center">
+        <input type="text" id="searchBar" placeholder="Search Stock">
+        <button id="seachBtn" type="submit"><i class="fa fa-search"></i></button>
+      </div>
+      <div class="row align-content-center">
+        <div class="view" data-view="home-view">
+          <p>No stock search yet</p>
+        </div>
+      </div>
+      <div class="view" data-view="search-view">
+        <div class="row align-content-center">
+          <h1 class="font-san-serif">Search Result</h1>
+        </div>
+        <div class="row">
+          <table>
+            <tbody class="displayTable">
+            </tbody>
+          </table>
+        </div>
+      </div>
+      <div class="view" data-view="stock-view">
+        <div class="row">
+          <div class="column-full align-content-space-between">
+            <h2 class="stock-name">Tesla Inc</h2>
+            <i id="close" class="fa fa-times"></i>
+          </div>
+        </div>
+        <div class="row">
+          <div class="column-full">
+            <h3 id="stock-current-price">759 &nbsp&nbsp +2.50 (0.33%)</h3>
+          </div>
+        </div>
+        <div class="row">
+          <div class="column-half ">
+            <div  class="align-content-space-between h3-margin-top-bottom">
+              <h3>Today's High</h3>
+              <h3 class="today-high">761.04</h3>
+            </div>
+            <div  class="align-content-space-between h3-margin-top-bottom">
+              <h3>Today's Low</h3>
+              <h3 class="today-low">761.04</h3>
+            </div>
+            <div  class="align-content-space-between h3-margin-top-bottom">
+              <h3>Today's Open</h3>
+              <h3 class="today-open">761.04</h3>
+            </div>
+            <div class="align-content-space-between h3-margin-top-bottom">
+              <h3>Previos Close</h3>
+              <h3 class="previos-close">761.04</h3>
+            </div>
```

Comment:
Rather than hardcoding in a number, I would just leave the numeric values here blank in the HTML, since you'll be replacing them anyway